Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM python:3.6.5-jessie
+
+env DEBIAN_FRONTEND noninteractive
+RUN echo 'deb http://deb.debian.org/debian jessie-backports contrib non-free main' >> /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y install --no-install-recommends \
+            build-essential \
+            chromedriver \
+            chrpath \
+            dbus-x11 \
+            ffmpeg \
+            libdbus-1-dev \
+            libdbus-glib-1-dev \
+            libespeak1 \
+            libffi-dev \
+            libfontconfig1 \
+            libfontconfig1-dev \
+            libfreetype6 \
+            libfreetype6-dev \
+            libnotify-cil-dev \
+            libssl-dev \
+            libxft-dev \
+            notification-daemon \
+            python3-dbus \
+            python-dbus \
+            python-dbus-dev \
+            python-imdbpy \
+            python-notify2 \
+            screenfetch \
+            sudo \
+            xdg-utils \
+            xorg  && \
+     rm -rf /var/lib/apt/lists/*
+
+
+WORKDIR /usr/local/share
+RUN wget -qO - https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 | tar xjf - && \
+  ln -s /usr/local/share/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/local/bin/phantomjs
+
+COPY . /app
```

Comment:
I'm not sure if this will set recursively, but you can set ownership during copy:
`COPY --chown=jarvis:jarvis . /app`

A big advantage is that since docker layers are read only, when copying first and then setting permission, you add twice the size of the content you copy. We noticed that when building image with pretty large content (>1GB) and when modifying ownership in another layer it added 1 GB in the image size. See https://medium.com/@lmakarov/the-backlash-of-chmod-chown-mv-in-your-dockerfile-f12fe08c0b55 and https://github.com/moby/moby/pull/34263#issuecomment-343963040